apps/opik-frontend/src/components/pages-shared/optimizations/MetricSettings/metricConfigs/LevenshteinMetricConfigs.tsx

```diff
@@ -47,7 +55,7 @@ const LevenshteinMetricConfigs = ({
             }
           />
           <Label htmlFor="case_sensitive" className="cursor-pointer text-sm">
-            Case sensitive
+            Case sensitive comparison
```

Comment:
This string repeats in multiple files

Export to const?

Reply:
We're going to change these later probably so I would keep it for now